addons/binding/pom.xml

```diff
@@ -28,6 +28,7 @@
     <module>org.openhab.binding.coolmasternet</module>
     <module>org.openhab.binding.dlinksmarthome</module>
     <module>org.openhab.binding.dscalarm</module>
+	<module>org.openhab.binding.dsmr</module>
```

Comment:
please replace tab by spaces

Reply:
@kaikreuzer our code guidelines suggest otherwise (tabs in xml) should we reformat this file once?

Reply:
For files that are not in the IDE workspace I actually do not care too much (they are not considered by our auto-formatting). They should just be in sync with the rest of the file, which does not seem to be the case here.